### PR TITLE
Update Mykrobe conda version from 0.7.0 to 0.9.0 

### PR DESCRIPTION
**Pull request by dankein**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Updating Mykrobe from Conda version 0.7.0 to 0.9.0

### CODE REVIEW

File: tools/mykrobe/macro.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<?xml version="1.1"?>
+<?xml version="1.2"?>
```
Comment (bgruening) [inline tools/mykrobe/macro.xml:1]: That one should stay with 1.1.

**Review by bgruening — COMMENTED** [review]
Thanks @dankein! One small comment.

**Review by bernt-matthias — COMMENTED** [review]
According to the release notes (https://github.com/Mykrobe-tools/mykrobe/releases) `mykrobe genotype removed. Same functionality is now available from mykrobe predict`

**Comment by bernt-matthias** [comment]
Superseded by https://github.com/galaxyproject/tools-iuc/pull/3849

**Pull request closed without merging**